framework/Optimizers/Optimizer.py

```diff
@@ -104,6 +104,7 @@ def __init__(self):
     self.optVarsInit['upperBound']      = {}                        # Dict containing upper bounds of each decision variables
     self.optVarsInit['lowerBound']      = {}                        # Dict containing lower bounds of each decision variables
     self.optVarsInit['initial']         = {}                        # Dict containing initial values of each decision variables
+    self.optVarsInit['ranges']          = {}                        # Dict containing initial values of each decision variables
```

Comment:
This comment is either incorrect, or needs some expansion to explain why it is different from self.optVarsInit['initial']